README.md

```diff
@@ -237,26 +237,28 @@ Append `-d` to run in detached mode. The first time you run a service, it is pro
 
 `sonarr` should be configurable the same as any other installation of it. Feel free to skip these steps if you know how to configure `sonarr` already.
 
-⚠️ It is critical that you use `transmission` instead of the IP address when configuring the download client, as well as `jackett` instead of the IP when setting up your indexer. This is because this uses docker-compose networking which means each service is accessible at the name of the service, rather than the host or even local IP address. If you use wireguard for `transmission` or `jackett` use `localhost` instead.
```

Comment:
Thanks for catching these.

Reply:
That was just my mistake ;) I didn't set a proper `SERVICE_NAMES` at the beginning for wireguard and couldn't use an aliases.